src/config/Config.zig

```diff
@@ -5032,6 +5032,12 @@ pub const Keybinds = struct {
             .{ .reset_font_size = {} },
         );
 
+        try self.set.put(
+            alloc,
+            .{ .key = .{ .unicode = 'j' }, .mods = .{ .shift = true, .ctrl = true, .super = true } },
```

Comment:
I picked this combination for parity with how taking a screenshot works on macOS. [Holding the <kbd>Control</kbd> key](
https://support.apple.com/en-us/102646#:~:text=Or%20press%20Control%20in%20addition%20to%20the%20other%20keys%20you%20press%20to%20take%20the%20screenshot.) picks the clipboard.

Reply:
I'm not sure if this needs a default keybinding but I see it also lines up with our other write screen back file actions so lets give it a go.